src/styles/themes/light.ts

```diff
@@ -34,7 +34,7 @@ const lightTheme = {
     successPressed: colors.greenPressed,
     transparent: colors.transparent,
     signInPage: colors.green800,
-    dangerSection: colors.tangerine800,
+    darkSupportingText: colors.darkSupportingText,
```

Comment:
this color key no longer exists, could you use `productDark800`?

Reply:
updated